src/sentry/suspect_resolutions/metric_correlation.py

```diff
@@ -0,0 +1,45 @@
+from datetime import timedelta
+
+from sentry import tsdb
+from sentry.models import Group
+
+
+def is_issue_error_rate_correlated(resolved_issue_id: int, issue2_id: int) -> bool:
```

Comment:
Would it make sense to change `resolved_issue_id` type to `Group`, and we pass in the resolved Group to this function instead of having to look it up again?